src/App.js

```diff
@@ -153,6 +124,7 @@ const render = ({
           >
             {modalState.component}
           </Modal>
+          <Toast {...toast}>{Object.keys(toast).length && toast.component}</Toast>
```

Comment:
do you need to check `Object.keys(toast).length`? toast should always be an object, and Toast will handle when it's closed I think?

Reply:
you're right, updated